presto-hive/src/main/java/com/facebook/presto/hive/SqlStandardAccessControl.java

```diff
@@ -194,6 +194,19 @@ public void checkCanGrantTablePrivilege(Identity identity, Privilege privilege,
         }
     }
 
+    @Override
+    public void checkCanRevokeTablePrivilege(Identity identity, Privilege privilege, SchemaTableName tableName)
+    {
+        if (checkTablePermission(identity, tableName, OWNERSHIP)) {
+            return;
+        }
+
+        HivePrivilege hivePrivilege = toHivePrivilege(privilege);
+        if (hivePrivilege == null || !metastore.hasPrivilegeWithGrantOptionOnTable(identity.getUser(), tableName.getSchemaName(), tableName.getTableName(), hivePrivilege)) {
+            denyGrantTablePrivilege(privilege.name(), tableName.toString());
```

Comment:
oh, I had the right method call in a later commit, but forgot to push it. Just did.